prisma/schema.prisma

```diff
@@ -0,0 +1,30 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+model Wallet {
+  pubkey      String       @id @unique
+  authority   String
+  memberShipType String
+  acceptSPL   Boolean
+  membership Membership[]
+  totalShares Int
+}
+
+model Membership {
+  memberPubkey   String 
+  
+  shareCount     Int
+  id             String @id @default(uuid())
+
+  wallet       Wallet @relation(references: [pubkey], fields: [walletPubkey])
+  walletPubkey String
+}
```

Comment:
If the primary key of `Wallet` is `(cluster, pubkey)`, the primary key of this would be `(cluster, walletPubkey, memberPubkey)`, we wouldn't need `id` in that case. The foreign key into `Wallet` would be `(cluster, walletPubkey)`, so the relation would be:

```prisma
wallet  Wallet  @relation(references: [cluster, pubkey], fields: [cluster, walletPubkey])
```

We would need a `cluster` field here obviously.

We also need a `Date`-type field here for the date the member was added on (you can check out the Figma design for the "Wallet Details" page where each member is shown with their addition date).

Reply:
yup id is quite redundant at the moment but was necessary to avoid errors whilst developing it, will also add a blockheight date so we can be more specific and have easier syncing with the on chain data